main.c

```diff
@@ -471,7 +488,7 @@ main(int argc, char **argv)
 			int kb = atoi(argv[0]);
 			bool found = false;
 			for (int cmp = 8; cmp <= 2048; cmp *= 2) {
-				if (kb == cmp)  {
+				if (kb == cmp)	{
```

Comment:
Thanks. I had the butterfly keyboard for a while. :(

Reply:
👍